core/src/main/java/org/mskcc/cbio/portal/dao/DaoGene.java

```diff
@@ -265,9 +278,10 @@ private static CanonicalGene getGene(long entrezGeneId) throws DaoException {
                     ("SELECT * FROM gene");
             rs = pstmt.executeQuery();
             while (rs.next()) {
+            	int geneticEntityId = rs.getInt("GENETIC_ENTITY_ID");
                 long entrezGeneId = rs.getInt("ENTREZ_GENE_ID");
                 Set<String> aliases = mapAliases.get(entrezGeneId);
-                CanonicalGene gene = new CanonicalGene(entrezGeneId,
+                CanonicalGene gene = new CanonicalGene(geneticEntityId, entrezGeneId,
```

Comment:
DaoGene may be internal, but we have endpoints like this: https://github.com/cBioPortal/cbioportal/blob/rc/web/src/main/java/org/cbioportal/web/GeneController.java which return a Gene object or list of Gene objects. If the Gene model adds geneticEntitiyId field, we need to suppress it before the response is delivered from the endpoint (maybe here: https://github.com/cBioPortal/cbioportal/blob/rc/web/src/main/java/org/cbioportal/web/mixin/GeneMixin.java)